docs/index.md

```diff
@@ -4,17 +4,31 @@
 
 # Welcome to Nightscout
 
+</br>
+
 ## Introduction
 
 Nightscout (CGM in the Cloud) is an open source, DIY project cloud application, which you can install yourself using free hosting from various cloud service providers. Once installed, Nightscout acts as a central repository of blood glucose and insulin dosing data for a single person, allowing you to view the CGM graph and other data anywhere using a web browser.
 
-The CGM data is sent to Nightscout using an uploader application such as xDrip+ or Spike, running on a mobile phone carried by the person being monitored. The application you'll need for uploading the data depends on which CGM device you're using.
+The CGM data is sent to Nightscout using an uploader application such as xDrip+, Spike and Medtronic uploader, running on a mobile phone carried by the person being monitored, or directly from your Dexcom or CareLink account. The application you'll need for uploading the data depends on which CGM device you're using.
```

Comment:
`or` instead of `and`:
```suggestion
The CGM data is sent to Nightscout using an uploader application such as xDrip+, Spike or Medtronic uploader, running on a mobile phone carried by the person being monitored, or directly from your Dexcom or CareLink account. The application you'll need for uploading the data depends on which CGM device you're using.
```

Reply:
Absolutely, and Dexcom share too. This is being documented and will be in the final version. Thanks

Reply:
Merged in latest PR